### ISSUE TITLE
Hugo Migration - Kubeadm Reference Docs don't show up correctly

### ISSUE THREAD

**Issue by jaredbhatti** [kubernetes/website#8267]
Kubeadm doesn't show up correctly in the left nav for Hugo. 

![screen shot 2018-05-01 at 12 04 22 pm](https://user-images.githubusercontent.com/6146946/39469307-d19beb92-4d37-11e8-8923-697920eab635.png)

**Comment by jaredbhatti** [comment]
Link: https://hugo-migration.docs.kubernetes.io/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase/

**Comment by bep** [comment]
The scripts that autogenerates docs into this site needs to be adjusted and re-generated. @zacharysarah has that on his TODO list. In this particular case they need front matter with a proper title. And probably also weight if specific order.

**Comment by bep** [comment]
@Bradamant3 you are right, my mistake. I see it now.

**Comment by bep** [comment]
I have added a temporary fix for this, but now my commit access is revoked, so I cannot push it.

This is, looking at it more closely, related to the missing adaptation of the generated docs.

* kubeadm is a section
* kubeadm/generated was not a section, hence it was put in the kudeadm section
* all the files in kubeadm/generated missed a title, hence it looked odd in the menu (blanks)
* I have added a _index.md file with toc_hide: true to the /generated folder, which makes it a section and hides it in the menu

But I cannot push this to the server because I have no push access.
